src/Okta.Sdk/Generated/IUserSchemaAttribute.Generated.cs

```diff
@@ -17,6 +17,14 @@ public partial interface IUserSchemaAttribute : IResource
     {
         string Description { get; set; }
 
+        IList<string> Enum { get; set; }
```

Comment:
Verify if this Enum should be a list of `IUserSchemaAttributeEnum` instead of a string.
cc @gormanho-okta @royalchan-okta